src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -71,9 +71,11 @@ System.Resources.Extensions.PreserializedResourceWriter</PackageDescription>
                       Lines="$(SuggestedBindingRedirectsPackageFileContent)"
                       Overwrite="true" />
 
-    <ItemGroup>
-      <Content Include="$(SuggestedBindingRedirectsPackageFile)"
-               PackagePath="buildTransitive\$(NetFrameworkMinimum)\$(PackageId).targets" />
-    </ItemGroup>
   </Target>
+
+  <ItemGroup Condition="'$(NetFrameworkMinimum)' != ''">
```

Comment:
>Condition="'$(NetFrameworkMinimum)' != ''" [](http://example.com/codeflow?start=13&length=42)

Does it make any difference checking `NetFrameworkMinimum` or checking the target framework to be NET Framework?